azure-mgmt-hanaonazure/azure/mgmt/hanaonazure/version.py

```diff
@@ -9,4 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.4.0"
+VERSION = "0.1.0"
```

Comment:
Can the version please be changed to "0.5.0"?

Reply:
yes, that's a bug in our generator, I'm used to have to fix that. It's fixed in the latest version of the generator but we cannot use it yet for some complex reasons